rest-server/core/src/main/scala/com/actionml/core/spark/SparkScaffoldEngine.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright ActionML, LLC under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * ActionML licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.actionml.core.spark
+
+import cats.data.Validated
+import com.actionml.core.engine.Engine
+import com.actionml.core.validate.ValidateError
+
+class SparkScaffoldEngine extends Engine {
+  private val lambdaAlgo = new SparkLambdaAlgorithm
+
```

Comment:
I don't see it this way, it should ideally be 

```
private val ldaAlgo = new LDAAlgorithm with SparkLambdaAlgorithm
```

this should make an implicit `val sc: SparkContext = ...` 

Also the parameters are in the engine's JSON and so already available as a string, so we may not need to pass them in to `Algo.train`.